ideascube/conf/kb_rwa_giz.py

```diff
@@ -0,0 +1,13 @@
+"""KoomBook conf"""
+from .kb import *  # noqa
+
+LANGUAGE_CODE = 'en'
+IDEASCUBE_NAME = 'GIZ'
```

Comment:
Did you know we don't need to specify the `IDEASCUBE_NAME` in the settings files any more? 😄